blender-mesh/src/lib.rs

```diff
@@ -93,12 +92,20 @@ impl BlenderMesh {
     }
 
     /// A map of material name to the material's data
-    pub fn materials(&self) -> &HashMap<String, PrincipledBSDF> {
+    pub fn materials(&self) -> HashMap<String, PrincipledBSDF> {
+        let mut res = HashMap::new();
+        for material in &self.materials {
+            res.insert(material.name.clone(), material.clone());
+        }
+        res
+    }
+
+    pub fn materials_vec(&self) -> &Vec<PrincipledBSDF> {
```

Comment:
You can just make this `materials` since this is already a breaking change anyway